tip-wlan/charts/wlan-prov-service/values.yaml

```diff
@@ -65,7 +65,9 @@ creds:
       password: DUMMY_POSTGRES_PASSWORD
```

Comment:
This is now duplicated. May be, if not in this PR, remove it and update charts to use the new properties added below.